core/block/editor/state/normalize.go

```diff
@@ -24,9 +28,17 @@ type Normalizable interface {
 }
 
 func (s *State) normalize(withLayouts bool) (err error) {
-	if err = s.Iterate(func(b simple.Block) (isContinue bool) {
```

Comment:
I would find useful to extract a few separate functions from this method:
something like 
normalizeSize
removeInvalidChildren
normalizeRows
normalizeLayouts

and made a unittest at least for this normalizeSize case

Reply:
Made normalize shorted and added test on normalizeSize